robot/EDA/tests/browser/contacts/verify_preferred_phone.robot

```diff
@@ -0,0 +1,187 @@
+*** Settings ***
+Documentation
+...     Verify Preferred Phone functionality
+
+
+Resource        robot/EDA/resources/EDA.robot
+Library         robot/EDA/resources/EDA.py
+Library         DateTime
+Library         cumulusci.robotframework.PageObjects
+...             robot/EDA/resources/ContactsPageObject.py
+...             robot/EDA/resources/TriggerHandlersPageObject.py
+Suite Setup     Run keywords
+...             Initialize test data
+...             Open test browser and print package details
+Suite Teardown  Capture screenshot and delete records and close browser
+
+*** Test Cases ***
+Verify basic preferred phone functionality
+    Select App Launcher App                 EDA
```

Comment:
while not necessary, you might want to consider changing `Select App Launcher App` to a [test setup](http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#test-setup-and-teardown). It doesn't really change how the test works, but it helps self-document the test. 

`[Setup]  Select App Launcher App  EDA`

Reply:
I moved these two calls into a Test Setup, so that they're called for each test.